IBX-3863: Configuration HTTP Client for Solr queries added (#1809)
ezsystems/developer-documentation: docs/search/solr_search_engine.md

![Solr Slave](solr.png)

## Configuring HTTP Client for Solr queries

> Solr uses Symfony HTTP Client to fetch and update Solr index. It's possible to configure timeout and number of max retries for that client using Solr Bundle's Semantic configuration:

Reviewer: Let's be here more specific
```suggestion
Ibexa Solr Bundle uses Symfony HTTP Client to fetch and update Solr index. It's possible to configure timeout and number of max retries for that client using Solr Bundle's Semantic configuration:
```
because Solr itself is a 3rd party app written in Java, it's our Ibexa integration/client which uses SF HTTP Client.

> Ibexa Solr Bundle uses Symfony HTTP Client to fetch and update Solr index. It's possible to configure timeout and number of max retries for that client using Solr Bundle's Semantic configuration:

Reviewer 2: ```suggestion
Ibexa Solr Bundle uses Symfony HTTP Client to fetch and update Solr index.
You can configure timeout and maximum number of retries for that client using Solr Bundle's Semantic configuration:
```
